src/ProcessMaker/Nayra/Contracts/Bpmn/ErrorEventDefinitionInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace ProcessMaker\Nayra\Contracts\Bpmn;
+
+/**
+ * ConditionalEventDefinition interface.
```

Comment:
The doc. comment must be corrected to ErrorEvent definition interface

Reply:
fixed